### ISSUE TITLE
Using dynamic links on social

### ISSUE THREAD

**Issue by idoadse** [EddyVerbruggen/nativescript-plugin-firebase#579]
Im trying to use dynamic links to achieve a mechanims similar to Firebase.invites, but on other sources (facebook, whatsapp, messenger and more).

- I'm currently implementing only android app (will later this month support iphones as well).

I have tried implementing it in the following way:
1. I have installed the following plugin https://www.npmjs.com/package/nativescript-social-share that supports sharing on social media
2. I have followed the firebase.Invites tutorial and added the following code to my androidmanifest.xml under activity:
			<intent-filter>
				<action android:name="android.intent.action.VIEW"/>
				<category android:name="android.intent.category.DEFAULT"/>
				<category android:name="android.intent.category.BROWSABLE"/>
				<data android:host="myawsomedomain.com" android:scheme="http"/>
				<data android:host="myawsomedomain.com" android:scheme="https"/>
			</intent-filter>
3. I have added callback to handle dynamic links on my firebase initializations: 
firebase,onDynamicLinkCallback: function (result) {
      console.log("Dynamic Link: " + result+ ", matchConfidence: "+ result); 
    }
4. I have also added a call to firebase.getInvitation() that logs its input
5. I have created a url (e.g. myawsomedomain.com/example) and sent it via whatsapp (and any other source) 
6. I have clicked the received link on the same device (device having the app installed)

I encounter the following symptoms:
- the app is opened succesfully
- no callback is called:
a) send invitation mentions no invitations is found
b) onDynamicLinkCallback is never triggered (I have also tried to register it on the firebase.init - not succesful either)

Debugging some of the plugin code yeild that there is no firebase dynamic link.

What is the way to achieve my requested flow? 
I have been looking at [this PR](https://github.com/EddyVerbruggen/nativescript-plugin-firebase/pull/524)  that I thought was supposed to solve the none-called callbacks issue, but update of my plugin to latest version seemed not to fix the issue.

I have also checked [this issue](https://github.com/EddyVerbruggen/nativescript-plugin-firebase/issues/503) but it seemed to happen there only on iOS.

If this is not supported for some reason maybe it will be possible to allow sendInvite to not open any contact list, but only return the url of the invite so I could send it myself.

Thanks for the help

**Comment by idoadse** [comment]
@EddyVerbruggen Do you have any inputs on this one? Thanks in advance

**Comment by manijak** [comment]
Yeah the `onDynamicLinkCallback` is not working on iOS. The app opens but the callback is not hit.
